src/util/Storage.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+#include <string>
+
+namespace YimMenu
+{
+	struct Storage
```

Comment:
This should be subject to change. I do not want to repeat the compile issues of YimMenu where each file that includes this has to be rebuilt. For now its okay.

Reply:
Then do the right thing and don't even let this file to exist.

Reply:
If this is not good for the long run, what do you guys suggest? I mean I don't see how it could be that much of an inconvenience but I didn't deal with this situation in YimMenu.

Reply:
I'm not a fan of a single file either, you could separate it into file categories at the least(SpoofingVariables.hpp etc)

Reply:
Okay, I will add a storage folder with separate files for different topics/features